pkg/core/storage/store.go

```diff
@@ -64,6 +64,8 @@ type (
 		// Key and value slices should not be modified. Seek can guarantee that key-value items are sorted by
 		// key in ascending way.
 		Seek(k []byte, f func(k, v []byte))
+		SeekFrom(k []byte, f func(k, v []byte))
```

Comment:
Maybe we can add a flag to public interface too, instead of having 3 functions?

Reply:
Fixed, but it might be kind of confusing because not all flags can be safely combined. So maybe for the public API it's still better to use three methods instead of flags.